spec/v1/envelope.md

```diff
@@ -19,5 +21,41 @@ standard DSSE fields.
     specifying its schema.
 -   `payload` MUST be a base64-encoded JSON [Statement].
 
+## File naming convention
+
+Envelopes SHOULD use the suffix `.json`.
+
+-   An Envelope containing an attestation about a particular SW supply chain
```

Comment:
Would it make any sense to qualify this with something like "If the attestation is being generated for use with the in-toto verifier..." and then have some other guidance for folks that are using them standalone?  Or may not know what step it's a part of?

Reply:
done.